.github/workflows/ci.yml

```
@@ -0,0 +1,58 @@
+      # 6. Docker 이미지 빌드 및 푸시 ( 캐시 옵션 추가)
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/loginWithJWT:latest
+          #  Docker 레이어 캐시 설정
+          cache-from: type=gha,mode=max
+          cache-to: type=gha,mode=max
+          # 이러면 변경된 부분만 새로 빌드함
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)